src/view/dialog/meldingInput/inputUtils.ts

```diff
@@ -16,29 +16,33 @@ export const betterErrorMessage = (error: FieldError, melding: string): FieldErr
     };
 };
 
-export const debounced = <T extends Function>(fn: T): { cleanup: () => void; invoke: T } => {
+export const debounced = <T extends Function>(fn: T): { isDirty: () => boolean; cleanup: () => void; invoke: T } => {
     let timer: any | undefined;
     const invoke = (...args: any): void => {
         clearTimeout(timer);
         timer = setTimeout(() => {
             fn(...args);
+            timer = undefined;
         }, 500);
     };
     const cleanup = () => {
         clearTimeout(timer);
     };
-    return { cleanup, invoke: invoke as unknown as T };
+    const isDirty = () => {
+        return timer !== undefined;
+    };
+    return { cleanup, invoke: invoke as unknown as T, isDirty };
 };
 
 export interface MeldingInputArgs {
     onSubmit: (e?: React.BaseSyntheticEvent) => Promise<void>;
     noeFeilet: boolean;
-    isSyncingKladd: boolean;
+    kladdErLagret: boolean;
```

Comment:
Jeg ønsket å vise et eget ikon for når man gjør selve lagringen (requesten er pending)